test/functional/tests/stress/test_kedr.py

```diff
@@ -208,3 +211,74 @@ def test_kedr_basic_io(module, unload_modules, install_kedr):
 
     with TestRun.step(f"Stopping kedr"):
         Kedr.stop()
+
+
+@pytest.mark.parametrize("module", cas_module.CasModule)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_kedr_basic_io_fs(module, unload_modules, install_kedr):
```

Comment:
Are you using 2nd and 3rd params? If not you can remove them in both tests

Reply:
Those are fixtures - they are executed before the test execution 😉

Reply:
Ok, I see. But now modules are unloaded in fixtures and loaded in test. Wouldn't it be more consistent to load them in fixtures?

Reply:
Modules have to be unloaded for kedr to start - that's why this is (unloading, startign kedr and then loading modules) being done in test body, not in fixture itself.